model/keyvalue_test.go

```diff
@@ -130,13 +130,13 @@ func TestKeyValueAsStringAndValue(t *testing.T) {
 	for _, tt := range testCases {
 		testCase := tt // capture loop var
 		t.Run(testCase.str, func(t *testing.T) {
-			assert.Equal(t, testCase.str, testCase.kv.AsString())
+			assert.Equal(t, testCase.str, testCase.kv.AsStringLossy())
```

Comment:
```suggestion
			assert.Equal(t, testCase.str, testCase.kv.AsString())
```

Reply:
This would not pass, the expected string is trucated